contracts/periphery/ZeroExTradeWrapper.sol

```diff
@@ -0,0 +1,81 @@
+/*
+    Copyright 2023 Index Cooperative
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.8.19;
+
+import { IERC20 } from "openzeppelin-contracts-4.9/token/ERC20/IERC20.sol";
+
+
+
+
+/**
+ * @title ZeroExTradeWrapper
+ * @author Index Coop
+ *
+ * Contract for wrapping ZeroEx trades
+ * Developed to fulfil calldata decoding requirments in the ledger integration
```

Comment:
nit: typos - fulfill, requirements

Reply:
let's also replace this description, since we have a function that moves funds it is for more than decoding

Reply:
Well this really is the main motivation why we are deploying this contract.  The 0x router already moves the funds, so this is not really an additional benefit that we get from using this wrapper contract over calling the 0x router directly.

The only additional benefit we get is enforcement of the tokenlimits instead of just trusting 0x to abide by it. I will add this to the description.